app/src/main/java/org/oppia/android/app/fragment/FragmentComponentImpl.kt

```diff
@@ -142,6 +143,7 @@ interface FragmentComponentImpl : FragmentComponent, ViewComponentBuilderInjecto
   fun inject(profileEditDeletionDialogFragment: ProfileEditDeletionDialogFragment)
   fun inject(profileEditFragment: ProfileEditFragment)
   fun inject(profileListFragment: ProfileListFragment)
+  fun inject(profileDeleteSuccessFragment: DeleteProfileSuccessDialogFragment)
```

Comment:
I'd expect the name here to be ``deleteProfileSuccessDialogFragment`` per the name of the class, and then this inject be moved up to be in alphabetical order.

Reply:
I have corrected this